chia/cmds/start_funcs.py

```diff
@@ -17,8 +17,18 @@
 
 def launch_start_daemon(root_path: Path) -> subprocess.Popen:
     os.environ["CHIA_ROOT"] = str(root_path)
-    # TODO: use startupinfo=subprocess.DETACHED_PROCESS on windows
-    process = subprocess.Popen([sys.argv[0], "run_daemon", "--wait-for-unlock"], stdout=subprocess.PIPE)
+    if sys.platform == "win32":
+        return subprocess.Popen(
+            [sys.argv[0], "run_daemon", "--wait-for-unlock"],
+            encoding="utf-8",
+            creationflags=subprocess.CREATE_NEW_PROCESS_GROUP | subprocess.CREATE_NO_WINDOW,
```

Comment:
I think the `if` can be used just to set a variable for the creation flags and then we can retain a single `Popen()` call since everything else is the same.

Reply:
Agreed and changed